regexes/device/mobiles.yml

```diff
@@ -7823,7 +7862,7 @@ Opsson:
 
 # orange
 Orange:
-  regex: 'SPV[ \-]?([a-z0-9]+)|Orange[ _-]([^/;]+) Build|SC/IHD92|FunTab ([^/;]+) Build'
+  regex: 'SPV[ \-]?([a-z0-9]+)|Orange[ _-]([^/;]+)( Build|\))(?<!Tahiti-LS\))|SC/IHD92|FunTab ([^/;]+) Build'
```

Comment:
Schould the term `(?<!Tahiti-LS\))` not be before the searched word, not behind?

Reply:
I tried to put in the beginning but it didn't work
success https://regex101.com/r/PaZaPd/1
bad https://regex101.com/r/PaZaPd/2

Reply:
I would write the regex this way: `Orange[ _-](?!Tahiti-LS\))([^\/;]+)( Build|\))`

Reply:
Thanks